apps/okreads/browser/src/app/app.component.html

```diff
@@ -23,7 +24,7 @@ <h1>okreads</h1>
     <div class="reading-list-container" data-testing="reading-list-container">
       <h2>
         My Reading List
-        <button mat-icon-button (click)="drawer.close()">
+        <button mat-icon-button aria-label="Close" (click)="drawer.close()">
```

Comment:
Close what? Would this be sufficient for a screen reader?

Reply:
Updated naming for aria attribute.